.storybook/gitignore

```diff
@@ -0,0 +1,27 @@
+/*
+!LICENSE
+!index.html
+!main.css
+!package.json
+!THIRD_PARTY_LICENSES
+!.gitignore
+!build
+!scripts
+!i18n
+!*.html
+!*.js
+!*.css
+!*.json
+!*.map
+!*.ico
+!*.png
+!*.jpg
+!*.jpeg
+!*.gif
+!*.svg
+!*.woff
+!*.woff2
+!*.ttf
+!*.eot
```

Comment:
how are you generating this list of extensions?

Reply:
Suggested from AI. Here's the actual list of extensions used:
```
$ pwd
box-ui-elements

$ ls -l | grep '\.' | rev | cut -d'.' -f1 | rev | sort | uniq
html
js
json
lock
map
md
svg
ts
txt
woff2
```